wordpress-mobile/WordPress-iOS
# ULS: update Auth to remove the Waiting for Google view from the navigation stack.

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/282
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/387

This uses the Auth changes that remove the Waiting for Google view when a successive view is displayed.

To test:

Signup: 
- `Continue with Google` from either login or signup.
- Select a Google account that has not been used to create a WP account.
- On the Signup Confirmation view, tap `Back`.
- Verify the Waiting for Google view does not appear.

Login with Password:
- `Continue with Google` from either login or signup.
- Select a Google account that has a WP password.
- On the Password view, tap `Back`.
- Verify the Waiting for Google view does not appear.

Login with 2FA:
- `Continue with Google` from either login or signup.
- Select a Google account that does not have a WP password, does have WP 2FA.
- On the 2FA view, tap `Back`.
- Verify the Waiting for Google view does not appear.

PR submission checklist:
